elements/src/main/java/org/vaadin/elements/elementui.js

```diff
@@ -33,7 +33,10 @@ window.org_vaadin_elements_ElementIntegration = function() {
 						var attributesToUpdate = boundAttributes[id][event];
 						for (var i = 0; i < attributesToUpdate.length; i++) {
 							var attribute = attributesToUpdate[i];
-							attributeChange = [+id, attribute, ""+ids[id][attribute]];
+							var valueToSend = ids[id][attribute];
+							attributeChange = [+id, attribute, valueToSend != null && typeof valueToSend === 'object' ?
```

Comment:
This lines gets quite long and difficult to read. Could you please split it up?

Reply:
Sure

Reply:
Done